hack/update-mockgen-dockerized.sh

```diff
@@ -0,0 +1,89 @@
+#!/usr/bin/env bash
+
+# Copyright 2022 Antrea Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -o errexit
+set -o nounset
+set -o pipefail
+set -o xtrace
+
+GOPATH=`go env GOPATH`
+ANTREA_PKG="antrea.io/antrea"
+
+function mockgen() {
```

Comment:
BTW, I am not totally sure why you are defining functions for these, but I have no objection

Reply:
hi, I provided the reason in the PR summary, the original purpose is to reduce the time when there is only mock codes need to be regenerated. The `codegen` will regenerate all clients codes which will take a much longer time locally.

Reply:
I understand the purpose of the PR and why you are introducing a new script, but not why you define bash functions (`mockgen`, `reset_year_change`). I't s a minor point.